operators/pkg/controller/elasticsearch/driver/default.go

```diff
@@ -46,7 +46,7 @@ import (
 // initContainerParams is used to generate the init container that will load the secure settings into a keystore
 var initContainerParams = keystore.InitContainerParameters{
 	KeystoreCreateCommand:         "/usr/share/elasticsearch/bin/elasticsearch-keystore create",
-	KeystoreAddCommand:            "/usr/share/elasticsearch/bin/elasticsearch-keystore add",
+	KeystoreAddCommand:            "/usr/share/elasticsearch/bin/elasticsearch-keystore add-file",
```

Comment:
Maybe `KeystoreAddFileCommand ` for explicitness?

Reply:
`InitContainerParameters` is used between ES, Kibana and APM Server, so we have to stay a bit abstract.

Reply:
You're right 👍